lib/chef/api_client.rb

```diff
@@ -1,7 +1,7 @@
 #
-# Author:: Adam Jacob (<adam@opscode.com>)
-# Author:: Nuo Yan (<nuo@opscode.com>)
-# Copyright:: Copyright (c) 2008 Opscode, Inc.
+# Author:: Adam Jacob (<adam@chef.io>)
+# Author:: Nuo Yan (<nuo@chef.io>)
+# Copyright:: Copyright (c) 2015 Chef Software, Inc.
```

Comment:
```Copyright:: Copyright (c) 2008,2015 Chef Software, Inc.```

I'm not sure exactly what relationship the old company has with the new one, but since we're not replacing the entire file the old copyright still applies.